### DISCUSSION TITLE
locale: do you have a plan about locale

### DISCUSSION THREAD

**Discussion by sixwaaaay** [containers/podman#11758] [Ideas]
In my view, locales can help podman's Promotion

**Comment by rhatdan** [comment]
It is an open source project, so if users want it translated into different locales then I believe we should do it.
How do we go about starting?

**Comment by sixwaaaay** [comment]
I think I can help translate the help document into Chinese to a certain extent, but I don't know how can I make a contribution. I see that the repository of minikube has a directory calls translations at the top level for locales. can podman do it like that?

**Reply by rhatdan** [reply]
Sure, just open a PR.  Are you talking about translating the man pages or all of the content?
